llvm/lib/Analysis/CtxProfAnalysis.cpp

```diff
@@ -29,6 +29,15 @@ cl::opt<std::string>
     UseCtxProfile("use-ctx-profile", cl::init(""), cl::Hidden,
                   cl::desc("Use the specified contextual profile file"));
 
+static cl::opt<CtxProfAnalysisPrinterPass::PrintMode> PrintLevel(
+    "ctx-profile-printer-level",
+    cl::init(CtxProfAnalysisPrinterPass::PrintMode::Everything), cl::Hidden,
```

Comment:
Why does it init to Everything here but JSON down on. line 178?

Reply:
When using in .ll tests, we get the whole dump by default. When used in unittests, it's used to get the json bit out.

I'm trying to see how I can move some unittests (the ICP one, mainly) to .ll.

If it's too weird, I can switch everything to one of the 2 cases and patch accordingly.